lib/events/dynamoevents/dynamoevents.go

```diff
@@ -60,6 +62,24 @@ type Config struct {
 	UIDGenerator utils.UID
 	// Endpoint is an optional non-AWS endpoint
 	Endpoint string `json:"endpoint,omitempty"`
+
+	// EnableContinuousBackups is used to enable PITR (Point-In-Time Recovery).
+	EnableContinuousBackups bool
+
+	// EnableAutoScaling is used to enable auto scaling policy.
+	EnableAutoScaling bool
+	// ReadMaxCapacity is the maximum provisioned read capacity.
+	ReadMaxCapacity int
+	// ReadMinCapacity is the minimum provisioned read capacity.
+	ReadMinCapacity int
```

Comment:
I'm still kinda curious why these new field don't have json tags on them - doesn't this struct basically map onto the file config (like its godoc suggests), so how do they look in the config?

IIRC the default behavior is that for json they marshal into the same name as the Go field name (so `ReadMaxCapacity: 1000`, etc.) if the json tag is not set.

Reply:
`AuditConfig` is the type that's read from file configuration, it's defined in types.proto:

https://github.com/gravitational/teleport/blob/master/lib/services/types.proto#L461-L480

That's then used to set `dynamoevents.Config`. The old json tags approach was probably just migrated away from and they were never removed. Which is why I didn't include them in any new code.

https://github.com/gravitational/teleport/blob/rjones/aws-fixes/lib/service/service.go#L923-L936